### DISCUSSION TITLE
Namecheap Education Domain + Digitalocean VPS Hosting

### DISCUSSION THREAD

**Discussion by rdhhauzan** [community/community#62558] [GitHub Education]
### Hi everyone,

As a title says, can i use a vps hosting from digitalocean or any other vendor to hosting a domain from Namecheap .me domain from Github Education?

**Comment by Awrsha** [comment]
Yes, you can absolutely use a VPS from DigitalOcean or another provider to host a .me domain from Github Education. Here are the basic steps:

Get a VPS - Sign up for a DigitalOcean droplet (their term for a VPS) or VPS from another provider. Make sure you choose the appropriate size based on your hosting needs.

Set up the VPS - Configure the operating system, install any software you need like a web server, set up firewall rules, etc. This is specific to the provider and OS you choose.

Get your .me domain - Purchase a .me domain from Namecheap as part of your Github Education benefits.

Point the domain to the VPS - You'll need to configure the domain's name servers (NS records) to point to the name servers of your VPS provider. This tells the domain where to send traffic.

Host your site - Upload your website files to the VPS and configure the web server. Your site will now be accessible at your .me domain name.

Secure your VPS - Make sure you set strong passwords, keep the OS and software updated, and implement any other security best practices.
